src/titiler/core/titiler/core/templates/wmts.xml

```diff
@@ -37,7 +37,7 @@
             <ows:Title>{{ title }}</ows:Title>
             <ows:Identifier>{{ layer_name }}</ows:Identifier>
             <ows:Abstract>{{ title }}</ows:Abstract>
-            <ows:WGS84BoundingBox crs="urn:ogc:def:crs:OGC:2:84">
+            <ows:WGS84BoundingBox crs={{ bounds_crs }}>
```

Comment:
would be weird to have `WGS84BoundingBox` not being in WGS84

Reply:
it is but I think this is just an artifact of the the WMTS spec, GDAL (via gdalinfo/qgis) seemed happy to work with a mars urn here in the crs. Unless its possible to call it a `BoundingBox` sans geoid name? I'll have to dig more into the wmts spec to see if there is a different choice to make.

Reply:
ahh looks like OGC 06-121r3 does mention an alternative `BoundingBox` that is more extendable (can be n dimensional for example). Now I can either update to only use `BoundingBox` or use `WGS84BoundingBox` if the bounds_crs is 4326. thoughts?